common/lib/common-utils/src/timer.ts

```diff
@@ -200,13 +200,20 @@ export class Timer implements ITimer {
         if (restart !== undefined) {
             // Restart with remaining time
             const remainingTime = this.calculateRemainingTime(restart);
-            this.startCore(remainingTime, () => restart.handler(), restart.duration);
-        } else {
-            // Run clear first, in case the handler decides to start again
-            const handler = this.runningState.handler;
-            this.clear();
-            handler();
+
+            if (remainingTime > 0)
+            {
```

Comment:
My bet is that the linter is complaining that this bracket is on the next line and that's why the ci is failing. @tylerbutler it's not immediately obvious the next step - `npm run prettier && npm run eslint` fails, what command should be run to fix it?

Reply:
Looking at the `package.json` file I would assume you would run `npm run format`

Reply:
`npm run prettier:fix` should do it

Reply:
(`lint:fix` script should work too; that way you only need to remember one thing)